### ISSUE TITLE
Warzone Pokémon do not have Original Trainers

### ISSUE THREAD

**Issue by Sensouu** [SmashMC-Development/Bugs-and-Issues#803]
**Describe the Bug**
This bug is a small one that becomes very inconvenient while trying to fill up the Pokedex. This bug is exactly as the title entails, Pokemon caught in Warzone do not have original trainers, at all, the section is left blank. That means while evolving Pokemon they'll never be dexed as you're not the OT, even if you personally captured it.  

**To Reproduce**
1. Go to Warzone and capture a Pokemon you need the evolution of for the Pokedex
2. Leave Warzone and claim your catches (original pokemon should be dexxed, Ot must change when it gets put into the WZ box)
3. Check OT, it should be blank.
4. Give the Pokemon enough levels to evolve. 
5. Check the dex and see its not dexed. 

**Servers with the Bug**
I only play on Reshiram but it probably exists with other servers as well. 

**Expected behavior**
The Pokemon should share the user who captured it's OT

**Screenshots**
![sasasddsd](https://user-images.githubusercontent.com/93282314/142078050-0de486e5-ac68-436a-a46d-8a316377ba5d.PNG)
Image above depicts a Cursola with A blank OT, level 91 and obtained through Warzone.
![ssaasasa](https://user-images.githubusercontent.com/93282314/142078051-dc69c25e-3e4c-4f77-bf47-2088bf2bb244.PNG)
This image depicts a Corsola caught through a Max Raid, level 63, with the OT as Sensouu. 

**Additional context**
If I captured a Pokemon because I need to evolve it, lets say, Weedle, if I need Beedrill for the dex it wouldn't be dexed as it doesn't share my OT

**Comment by Mikirae** [comment]
Do you know when exactly you captured this Pokemon?
And are you saying you captured a Pokemon and it had an OT, and then you leveled it up until it evolved and the OT is lost?

**Comment by Sensouu** [comment]
> Do you know when exactly you captured this Pokemon?
> And are you saying you captured a Pokemon and it had an OT, and then you leveled it up until it evolved and the OT is lost?

The Pokémon used is quite old, I think, but even with newer Pokémon they’re lacking an OT. I can catch some Pokémon tomorrow to see if this is still happening. I only realized it while trying to dex some gen 1 evolutions (weezing, muk, kakuna, shelder). And them not showing up in my dex despite me vividly remembering capturing them. 
the OT is never there to begin with, while taking my screenshots I checked some legendary Pokémon and the OT was still blank, but since they don’t evolve it was dexed fine (except maybe Solgaleo? But I’ll cross that bridge when I get to it.) 
Maybe my game could just be having a bad day and I’m jumping to conclusions, but my username hasn’t changed from when I captured the exampled Pokémon and now
